examples/stable-diffusion/text_to_image_generation.py

```diff
@@ -585,6 +607,12 @@ def main():
         else:
             outputs = pipeline(prompt=args.prompts, **kwargs_call)
 
+    if args.optimize and quant_config_path and config.measure:
+        from neural_compressor.torch.quantization import finalize_calibration
```

Comment:
does --optimize must be added if we are using FP8?

Reply:
Is this optimize flag only used during measurement??  It's quite confusing, can you also give more description or better name perhaps?

Reply:
Hi @jiminha , --optimize argument is for running the optimized version of sdxl pipeline. currently we are supporting both bf16 and fp8 under this and hence measurement is also included under this flag. please note default sdxl pipeline doesn't support fp8 in this repo.